core/integration/features/spot/accounts_and_fees/trading_fees_spot.feature

```diff
@@ -0,0 +1,172 @@
+      | party2 | BTC/ETH   | sell | 100    | 9000  | 0                | TYPE_LIMIT | TIF_GTC | party-order12   |
+    #During opening auction, holding asset is transfered from general account to holding account: 0080-SP-AUC-001
+    Then "party1" should have holding account balance of "4000" for asset "ETH"
+    Then "party1" should have general account balance of "6000" for asset "ETH"
+    Then "party2" should have holding account balance of "200" for asset "BTC"
+
+    And the parties amend the following orders:
+      | party  | reference    | price | size delta | tif     |
+      | party2 | party-order2 | 1000  | 0          | TIF_GTC |
+
+    And the opening auction period ends for market "BTC/ETH"
+    And the market data for the market "BTC/ETH" should be:
+      | mark price | trading mode            | horizon | min bound | max bound |
+      | 1500       | TRADING_MODE_CONTINUOUS | 360000  | 976       | 2268      |
+
+    And the following trades should be executed:
+      | buyer  | price | size | seller |
+      | party1 | 1500  | 100  | party2 |
+    Then the network moves ahead "10" blocks
+
+    Then "party1" should have holding account balance of "2000" for asset "ETH"
+    Then "party1" should have general account balance of "6500" for asset "ETH"
+    Then "party1" should have general account balance of "100" for asset "BTC"
+
+    Then "party2" should have holding account balance of "100" for asset "BTC"
+    #party2 sold 1BTC for 15ETH to party1
+    Then "party2" should have general account balance of "300" for asset "BTC"
+    Then "party2" should have general account balance of "1500" for asset "ETH"
+
+    Then the order book should have the following volumes for market "BTC/ETH":
+      | side | price | volume |
+      | buy  | 1000  | 200    |
+      | sell | 9000  | 100    |
+
+    And the parties amend the following orders:
+      | party  | reference     | price | size delta | tif     |
+      | party2 | party-order12 | 1000  | 0          | TIF_GTC |
+
+    And the following trades should be executed:
+      | buyer  | price | size | seller |
+      | party1 | 1000  | 100  | party2 |
+
+    Then the market data for the market "BTC/ETH" should be:
+      | mark price | trading mode            | auction trigger             |
+      | 1500       | TRADING_MODE_CONTINUOUS | AUCTION_TRIGGER_UNSPECIFIED |
+
+    Then the following transfers should happen:
+      | from   | to     | from account            | to account                       | market id | amount | asset |
+      | party1 | market | ACCOUNT_TYPE_GENERAL    | ACCOUNT_TYPE_FEES_MAKER          | BTC/ETH   | 10     | ETH   |
+      | party1 | market | ACCOUNT_TYPE_GENERAL    | ACCOUNT_TYPE_FEES_INFRASTRUCTURE | BTC/ETH   | 30     | ETH   |
+      | market | party1 | ACCOUNT_TYPE_FEES_MAKER | ACCOUNT_TYPE_GENERAL             | BTC/ETH   | 10     | ETH   |
```

Comment:
it's just because of the mechanics I explained to your other general misunderstanding about fees. It's a bit ugly but it's how it's implemented. 

party 2 is a seller, they are the aggressor. They don't have nor expected to have a general account in the quote balance. Party 1 is the buyer, they're not *really* paying the fees. Lets say that cost of the trade is 1000, and the fees on that are 10, then party1 pays 9990 to party2, then pays 10 to the fees. Party1 paid exactly how much the trade cost and paid not fees, and party 2, the aggressor got 9990, i.e. paid 10 in fees. 

as party1 was the maker on this trade so they get the maker fee back. 

I guess this could be made less ugly by just not making the make fee transfer, but this will also be slightly confusing so not sure what's better.

Reply:
But what's stopping from first transferring everything to party2 (10ETH for 1 BTC) and then paying their fees (0.4ETH) from that account? That'd keep the transfers clean and probably save many raised eyebrows and user queries in the future?

Reply:
I agree with Witolds suggestions if its an easy change 👍  I think cleaner and a little easier to trace back, I'm not sure what TransferType that first transfer of 10  from party1 to party2 is using but it's probably misleading right?

Reply:
ok, I'm perfectly fine with making the full payment from the buyer to the seller and then making the fee payment from the seller in this case. @cdummett this has not functional meaning, but it affects what the ledger looks like, i.e. as in the PR right now it looks like:
Given the seller (party2) is the aggressor and the buyer is party1 (only the quote asset transfers):
* party1 transfers the trade amount - fees to party2 
* party1 transfers the fees to the fees accounts

as opposed to what's @witgaw suggesting which is:
* party1 transfers the trade amount to party2
* party2 transfers the fees to the fee accounts

I think questions gonna be asked either way but I'm happy to change this to the latter.